ports/steamworks-sdk/vcpkg.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "steamworks-sdk",
+  "version": "1.60",
+  "description": "Steamworks is a free collection of tools available to all developers for use in their games or software on Steam.",
+  "homepage": "https://partner.steamgames.com/doc/sdk",
+  "license": null,
+  "supports": "((windows & x64) | (windows & x86) | (osx & x64) | (linux & x64) | (linux & x86)) & !static & !uwp",
```

```suggestion
  "supports": "!static & !uwp & !arm",
```
Does this port support mingw and arm？